tools/rename_samples.cwl

```diff
@@ -0,0 +1,63 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: cellranger_count
+doc: "Run cellranger count on a set of fastq files"
+
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/cellranger:6.0'
+  - class: ResourceRequirement
+    ramMin: 20000
+    coresMin: 16
+  - class: InlineJavascriptRequirement
+  - class: LoadListingRequirement
+    loadListing: shallow_listing
+
+baseCommand: [mkdir, -p]
+
+arguments:
+  - position: 1
+    shellQuote: false
+    valueFrom: >-
+     $(inputs.sample_name) &&
+     ${
+      var cp_cmd = "";
```

Comment:
Looks good - I'd try maybe `mv` instead of `cp` since they are on the same filesystem. In practice, doesn't seem to be too bad, but the larger the input file, the more unnecessary overhead `cp` might make

Reply:
Like, basically in your IDE in this script, replace `cp` with `mv` and give it a go, so hopefully pretty painless. You can also feel free to cut task task early if it seemed to do the expected renaming before counts runs